app/models.py

```diff
@@ -961,9 +959,11 @@ def query_records_accessible_by(
             )
 
         logic = getattr(cls, mode)
-        return logic.query_accessible_rows(
-            cls, user_or_token, columns=columns
-        ).options(options)
+        query = logic.query_accessible_rows(cls,
+                                            user_or_token, columns=columns)
+        for option in options:
+            query = query.options(option)
```

Comment:
Why the for loop instead of, say, `query.options(options)`?

Reply:
I found options wanted a single option and not a list when using select.

Reply:
But, but... https://docs.sqlalchemy.org/en/14/orm/query.html#sqlalchemy.orm.Query.options

Reply:
I see, maybe ** instead of just a list then?

Reply:
Isn't options a list?

Reply:
[16:35:36 basehandler] Error response returned by [/api/internal/recent_gcn_events]: [ExecutionOption Core or ORM object expected, got [<sqlalchemy.orm.strategy_options._UnboundLoad object at 0x1676b3d00>].]

when passing it a list.

Reply:
🤷‍♂️